quaint/src/connector/mysql/conversion.rs

```diff
@@ -337,16 +328,6 @@ impl TakeRow for my::Row {
                         return Err(Error::builder(kind).build());
                     }
                 },
-                #[cfg(not(feature = "chrono"))]
-                typ => {
-                    let msg = format!(
-                        "Value of type {:?} is not supported with the current configuration",
-                        typ
-                    );
-
-                    let kind = ErrorKind::conversion(msg);
-                    Err(Error::builder(kind).build())?
-                }
```

Comment:
This is worth double-checking.
However, since this PR effectively acts as if the `chrono` feature was applied 100% of the time, it should be safe to remove.